Database/Corrections/cataQuestFixes.lua

```diff
@@ -2962,6 +2962,9 @@ function CataQuestFixes.Load()
         [25467] = { -- Kliklak's Craw
             [questKeys.startedBy] = {{40276},nil,{54345}},
         },
+        [25469] = { -- Forces of Nature: Mountain Giants
+            [questKeys.objectives] = {nil,nil,nil,nil,{{{40026},40026,"Mountain Giants Rallied",Questie.ICON_TYPE_INTERACT}}},
```

Comment:
A `killCreditObjective` with only a single ID does not make sense, over using a simple "monster" objective

Reply:
https://github.com/Questie/Questie/pull/6255/commits/18fcf433739b13bd5027cc3de716a350d90938e8